src/vcpkg/platform-expression.cpp

```diff
@@ -543,7 +543,19 @@ namespace vcpkg::PlatformExpression
                         case Identifier::uwp:
                             return true_if_exists_and_equal("VCPKG_CMAKE_SYSTEM_NAME", "WindowsStore");
                         case Identifier::xbox:
-                            // This identifier is intended to be enabled via VCPKG_DEP_INFO_OVERRIDE_VARS
+                            if (true_if_exists_and_equal("VCPKG_CMAKE_SYSTEM_NAME", ""))
+                            {
+                                auto is_xbox = context.find("VCPKG_TARGET_IS_XBOX");
```

Comment:
I think the signal we agreed upon is `VCPKG_XBOX_CONSOLE_TARGET`?

The `VCPKG_TARGET_IS_Xxx` variables are outputs, not inputs, and I think all get set by vcpkg_common_definitions today?

Reply:
> and I think all get set by vcpkg_common_definitions today?

Yes they commonly do, however I would also consider them being set be the triplet itself if necessary and not generally appropriate for `vcpkg_common_definitions`. Especially with overlay triplets/ports you might want to add something like this if it doesn't yet exist.

Reply:
I think it is more than reasonable for the things supported in the main vcpkg registry that the tool knows about such things, and that it is not reasonable to hold working support for platforms we know and care about for hypothetical things that don't 'yet exist'.